python/paddle/fluid/framework.py

```diff
@@ -1300,6 +1439,26 @@ def __init__(self):
         self._current_role = core.op_proto_and_checker_maker.OpRole.Forward
         self._op_role_var = []
 
+    def equal(self, other):
```

Comment:
Please use Python style __eq__ method

https://docs.python.org/3/reference/datamodel.html#object.__eq__

or move this method to unittest?

Reply:
OK!Thanks!